clients/da-vinci-client/src/main/java/com/linkedin/davinci/VersionBackend.java

```diff
@@ -98,6 +102,13 @@ public class VersionBackend {
     this.compressor = Lazy.of(
         () -> backend.getCompressorFactory().getCompressor(version.getCompressionStrategy(), version.kafkaTopicName()));
     backend.getVersionByTopicMap().put(version.kafkaTopicName(), this);
+    long daVinciPushStatusCheckIntervalInMs = this.config.getClusterProperties()
```

Comment:
So we don't want to use a `getter` here?

Reply:
I didn't realize that config inherits VeniceServerConfig; changed to use `getter` function.